server/setup.py

```diff
@@ -7,6 +7,8 @@
 # Django 1.8.0 requires Python >= 2.7
 if PYTHON_MAJOR_MINOR < '2.7':
     DJANGO_REQUIRES = 'django >=1.4.0, <1.8.0'
+elif PYTHON_MAJOR_MINOR < '3.0':
+    DJANGO_REQUIRES = 'django>=1.4.0, <2.0.0'
```

Comment:
Since Pulp 2 is Python 2-only, I think you can just add this under the "else" instead.

Reply:
Although it's interesting that python 2 is attempting to install a python 3-only package at all.  Django should have their ```python_requires``` set to avoid this issue